In-house patch to remove the castellan.tests.functional.config from the
oslo.config.opts entry point since tests are not delivered.
This patch is not suitable for pushing upstream.

--- castellan-0.4.0/setup.cfg.~1~	2016-08-30 02:32:30.279797199 -0800
+++ castellan-0.4.0/setup.cfg	2016-08-30 02:34:06.069242220 -0800
@@ -24,7 +24,6 @@ packages =
 
 [entry_points]
 oslo.config.opts = 
-	castellan.tests.functional.config = castellan.tests.functional.config:list_opts
 	castellan.config = castellan.options:list_opts
 
 [build_sphinx]
